api/application.go

```diff
@@ -31,7 +31,7 @@ type ApplicationAPI struct {
 
 // Application Params Model
 //
-// Params allowed to create or update Applications
+// # Params allowed to create or update Applications
```

Comment:
Please revert all the backend changes related to golint, this should be done in a separate PR.